### PR TITLE
add missing headers

### PR DESCRIPTION
**Pull request by sslattery**

### CODE REVIEW

**Review by streeve — APPROVED** [review]

**Comment by terwin** [comment]
@sslattery what about `Picasso_UniformCartesianMeshMapping.hpp`?

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -41,7 +41,7 @@ add_library(picasso ${SOURCES})
 
 target_link_libraries(picasso
   Kokkos::kokkos
-  Kokkos::kokkoskernels
+  kokkoskernels
```
Comment (dalg24) [inline src/CMakeLists.txt:44]: ?

Comment (sslattery) [reply src/CMakeLists.txt:44]: ah crap missed this - this is part of my Trilinos build hack. I need to do a PR that detects Trilinos or something and just switches these so we can avoid this.

Comment (dalg24) [reply src/CMakeLists.txt]: You could create an alias if it the target does not exist

**Pull request merged**